app/src/main/java/com/sun/heartrate/data/database/HeartDatabase.kt

```diff
@@ -1,5 +1,98 @@
 package com.sun.heartrate.data.database
 
-class HeartDatabase {
+import android.content.ContentValues
+import android.content.Context
+import android.database.Cursor
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+import com.sun.heartrate.data.model.HeartModel
+
+class HeartDatabase(context: Context) : SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION) {
```

Comment:
```suggestion
class HeartDatabase(
context: Context
) : SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION) {
```
trong kotlin convention
cách viết class ntn em nhé